src/Adapter/Presenter/Product/ProductPresenter.php

```diff
@@ -74,14 +75,20 @@ class ProductPresenter
      */
     protected $translator;
 
+    /**
+     * @var EmbeddedAttributesFilter
+     */
+    private $embeddedAttributesFilter;
+
     public function __construct(
         ImageRetriever $imageRetriever,
         Link $link,
         PriceFormatter $priceFormatter,
         ProductColorsRetriever $productColorsRetriever,
         TranslatorInterface $translator,
         HookManager $hookManager = null,
-        Configuration $configuration = null
+        Configuration $configuration = null,
+        EmbeddedAttributesFilter $embeddedAttributesFilter = null
```

Comment:
Done this way to avoid adding a new mandatory constructor argument which would be a BC break